web/src/buddy_list.ts

```diff
@@ -393,13 +419,17 @@ export class BuddyList extends BuddyListConf {
     }
 
     update_section_header_counts(): void {
-        const {total_human_subscribers_count, other_users_count} = this.render_data;
+        const {other_users_count} = this.render_data;
```

Comment:
`non_participant_subscribers_count` should be in the `render_data` already right?